com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md

```diff
@@ -91,6 +92,10 @@ These settings control lighting features for your rendering components. Here you
 | **Metallic Indirect Fallback**     | Enable the checkbox to render Materials with base color as diffuse for this Camera. This renders metals as diffuse Materials. This is a useful Frame Setting to use for real-time Reflection Probes because it renders metals as diffuse Materials to stop them appearing black when Unity can't calculate several bounces of specular lighting. |
 | **Sky Reflection**                 | Enable the checkbox to allow this Camera to use the Sky Reflection. The Sky Reflection affects specular lighting. |
 | **Direct Specular Lighting**       | Enable the checkbox to allow this Camera to render direct specular lighting. This allows HDRP to disable direct view dependent lighting. It doesn't save any performance. |
+| **Subsurface Scattering** | Enable the checkbox to make HDRP compute Screen Space Subsurface Scattering. |
+| - **Quality Mode**        | Use the drop-down to select how the quality level for Subsurface Scattering is set.<br /> **From Quality Settings**: The Camera uses the Subsurface Scattering **Sample Budget** property that is set in the Material Section of your Unity Project's **Quality Settings**.<br /> **Override Quality Settings**: Allows you to set a custom sample budget for sample surface scattering for this Camera in the field below.|
```

Comment:
After the line breaks, can you add a bullet point( &#8226; )? So
<br/> &#8226;

Reply:
Done! Also it turns out we already had an entry for Subsurface Scattering, but it the wrong place/order and without the extra fields for quality/samples, so I fixed this issue too.